src/signin-signup/signup-form/handleSignUp.ts

```diff
@@ -0,0 +1,26 @@
+import axiosInstance from "@/lib/axios";
+
+interface handleSignUpProps {
+  email: string;
+  password: string;
+}
+
+const handleSignUp = async ({
+  email,
+  password,
+}: handleSignUpProps): Promise<boolean> => {
+  return await axiosInstance
+    .post("/sign-up", {
+      email: email,
+      password: password,
+    })
```

Comment:
프로퍼티명과 값이 같다면 이렇게 쓸 수 있습니다.
```suggestion
    .post("/sign-up", {
      email,
      password,
    })
```